crates/api_common/src/utils.rs

```diff
@@ -245,6 +247,16 @@ pub fn check_post_deleted_or_removed(post: &Post) -> Result<(), LemmyError> {
   }
 }
 
+pub fn check_comment_depth(comment: &Comment) -> Result<(), LemmyError> {
+  let path = &comment.path.0;
+  let length = path.split(".").collect::<Vec<&str>>().len();
+  if length > MAX_COMMENT_DEPTH_LIMIT {
+    Err(LemmyError::from_message("max_comment_depth_reached"))
+  } else {
+    Ok(())
+  }
+}
```

Comment:
Move this into `crates/api_crud/src/comment/create.rs` if its only used there.

Reply:
Done